modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/Select/Select.tsx

```diff
@@ -229,11 +255,13 @@ const Main = ({
 						name={`${name}_field`}
 						onChange={onChange}
 						options={normalizedOptions}
+						onSelectionChange={onSelectionChange}
+						placeholder={placeholder}
 						predefinedValue={newPredefinedValue}
 						readOnly={readOnly}
 						required={otherProps.required}
-						selectedKey={selectedKey || (newValue as string)}
-						showEmptyOption={false}
+						selectedKey={selectedKey ??	 (newValue as string)}
```

Comment:
check if this won't cause the same bug mentioned before

Reply:
Hi @rebsilva. It's working fine.

[Select with keyboard.webm](https://github.com/liferay-forms/liferay-portal/assets/26491796/bb1fbc46-1315-4f78-ab82-0195b002077c)

Reply:
awesome @alinedoleron! :purple_heart: